charts/lfx-v2-project-service/templates/ingressroute.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright The Linux Foundation and each contributor to LFX.
+# SPDX-License-Identifier: MIT
+---
+apiVersion: traefik.io/v1alpha1
+kind: IngressRoute
+metadata:
+  name: lfx-v2-project-service
+  namespace: lfx
+spec:
+  entryPoints:
+    - web
+    - websecure
+  routes:
+    - kind: Rule
+      match: >-
+        Host(`{{.Values.ingress.hostname}}`)
+      priority: 10
+      # TODO: add heimdall middleware once it is working - currently it causes a 403
```

Comment:
Suggestion: create a flag for enabling/disabling via the values file, so that the chart can be used standalone or as part of the umbrella chart. Even the inclusion of the ingress-route (or gateway API http route) could/should be configurable IMO